reimbursement.md

```diff
@@ -220,44 +222,28 @@ Example `amount` set to `2.55` and custom `description` to `Earned Stars`.
 
 ```sh
 curl -X POST \
-  https://api.fidel.uk/v1/transactions/{transactionId}/reimbursement \
+  https://api.fidel.uk/v1/transactions/{transactionId}reimbursement \
```

Comment:
I think we're missing a `/` 😁

Reply:
thank you, good catch fixed here: https://github.com/FidelLimited/docs/pull/207/commits/44b7b13e5b6ec37498a3fe1b50d938b2730e570c